### PR TITLE
Gpres productivity

### PR DESCRIPTION
**Pull request by darcatron**
Making Pull Requests Great Again!

### CODE REVIEW

File: swal2.js [hunk 1]
```diff
@@ -0,0 +1,877 @@
...
+  window.swalClasses = {
+    modal: 'sweet-alert',
+    overlay: 'sweet-overlay',
+    confirm: 'sweet-confirm',
+    cancel: 'sweet-cancel'
+  };
+
+  var mediaqueryId = 'sweet-alert-mediaquery';
+  var alertTypes   = ['error', 'warning', 'info', 'success'];
+  var defaultParams = {
```
Comment (darcatron) [inline swal2.js]: I am a comment

File: swal2.js [hunk 2]
```diff
@@ -0,0 +1,877 @@
...
+    buttonsStyling: true,
+    reverseButtons: false,
+    imageUrl: null,
+    imageWidth: null,
+    imageHeight: null,
+    imageClass: null,
+    timer: null,
+    width: 500,
+    padding: 20,
+    background: '#fff'
```
Comment (darcatron) [inline swal2.js]: Another comment about how good FFF is as a color

Comment (darcatron) [inline swal2.js]: True^

File: swal2.js [hunk 3]
```diff
@@ -0,0 +1,877 @@
...
+    var newClass = ' ' + elem.className.replace(/[\t\r\n]/g, ' ') + ' ';
+    if (hasClass(elem, className)) {
+      while (newClass.indexOf(' ' + className + ' ') >= 0) {
+        newClass = newClass.replace(' ' + className + ' ', ' ');
+      }
+      elem.className = newClass.replace(/^\s+|\s+$/g, '');
+    }
+  };
+
+  var escapeHtml = function(str) {
```
Comment (darcatron) [inline swal2.js]: Sneaky

File: swal2.js (hunk 2 above)
Comment (sdenee01) [inline swal2.js]: I disagree.

File: swal2.js [hunk 4]
```diff
@@ -0,0 +1,877 @@
...
+  var mediaqueryId = 'sweet-alert-mediaquery';
+  var alertTypes   = ['error', 'warning', 'info', 'success'];
+  var defaultParams = {
+    title: '',
+    text: '',
+    html: '',
+    type: null,
+    animation: true,
+    allowOutsideClick: true,
+    allowEscapeKey: true,
```
Comment (darcatron) [inline swal2.js]: sneaky

File: swal2.js (hunk 2 above)
Comment (darcatron) [inline swal2.js]: bless up, that's a beautiful color

File: swal2.js [hunk 5]
```diff
@@ -0,0 +1,877 @@
...
+      var e = event || window.event;
+      var target = e.target || e.srcElement;
+      var targetedConfirm = hasClass(target, window.swalClasses.confirm);
+      var targetedCancel  = hasClass(target, window.swalClasses.cancel);
+      var modalIsVisible  = hasClass(modal, 'visible');
+
+      switch (e.type) {
+        case 'mouseover':
+        case 'mouseup':
+        case 'focus':
```
Comment (darcatron) [inline swal2.js]: :100:

File: swal2.js (hunk 2 above)
Comment (sdenee01) [inline swal2.js]: another one

File: swal2.js [hunk 6]
```diff
@@ -0,0 +1,877 @@
+// SweetAlert2
+// github.com/limonte/sweetalert2
```
Comment (darcatron) [inline swal2.js]: bless up

File: README.md [hunk 7]
```diff
@@ -1 +1,2 @@
-# GPRES
```
Comment (darcatron) [inline README.md]: test

File: README.md [hunk 8]
```diff
@@ -1 +1,2 @@
...
\ No newline at end of file
+# GPRES
```
Comment (darcatron) [inline README.md]: bless up

Comment (sdenee01) [inline README.md]: another one

**Pull request closed without merging**